pages/skill.php

```diff
@@ -278,7 +278,7 @@ protected function generateContent()
             {
                 $this->addJS('?data=zones&locale='.User::$localeId.'&t='.$_SESSION['dataKey']);
 
-                $trainer = new CreatureList(array(CFG_SQL_LIMIT_NONE, ['ct.id', $list], ['s.guid', NULL, '!'], ['ct.npcflag', 0x10, '&']));
+                $trainer = new CreatureList(array(CFG_SQL_LIMIT_NONE, ['ct.ID', $list], ['s.guid', NULL, '!'], ['ct.npcflag', 0x10, '&']));
```

Comment:
Oh yeah, right. That got mixed up. I thoght on the first sight it was referencing npc_trainer...